src/empty.js

```diff
@@ -1 +1 @@
-export default {};
+module.exports = {};
```

Comment:
Rollup doesn't work with commonjs

Reply:
when resolveId be called, the context module should be a commonjs module, so just return a commonjs replacement.

Reply:
Why? Commonjs doesn't make any sense for rollup. It's just a code without any exports.